glean/src/core/context.ts

```diff
@@ -45,6 +47,13 @@ export class Context {
   private _initialized = false;
   // Whether or not Glean is in testing mode.
   private _testing = false;
+  // A map of metric types and their constructors.
+  // This map is dinamically filled everytime a metric type is constructed.
+  //
+  // If a metric is not on this map it cannot be serialized from the database.
```

Comment:
```suggestion
  // If a metric is not in this map it cannot be serialized from the database.
```

Also it should be "deserialized", shouldn't it?

Reply:
```suggestion
  // If a metric is not in this map it cannot be deserialized from the database.
```